src/Catalog.Api/Mappers/ItemMapper.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Catalog.Api.Requests;
+using Catalog.Api.Responses;
+
+namespace Catalog.Api.Mappers
+{
+    public static class ItemMapper
+    {
+        public static Domain.Entities.Item Map(ItemRequest request, int? itemId = null)
+        {
+            return new Domain.Entities.Item
+            {
+                Id = itemId ?? 0,
+                Name = request.Name,
+                Description = request.Description,
+                ImageUrl = request.ImageUrl,
+                Category = new Domain.Entities.Category
+                {
+                    Id = request.CategoryId
+                },
+                Price = request.Price,
+                Amount = request.Amount
+            };
+        }
+
+        public static ItemResponse Map(Domain.Entities.Item item)
+        {
+            return new ItemResponse
+            {
+                Id = item.Id,
+                Name = item.Name,
+                Description = item.Description,
+                ImageUrl = item.ImageUrl,
+                CategoryId = item.Category.Id,
+                Price = item.Price,
+                Amount = item.Amount,
+                Links = new List<Link>
+                {
+                    new Link
+                    {
+                        Href = $"/catalog/item/{item.Id}",
+                        Rel = "update_item",
+                        Method = "PUT"
+                    },
+                    new Link
+                    {
+                        Href = $"/catalog/item/{item.Id}",
+                        Rel = "delete_item",
+                        Method = "DELETE"
+                    },
+                    new Link
+                    {
+                        Href = $"/catalog/item/{item.Category.Id}{{?pageSize,page}}",
```

Comment:
correct me if I'm wrong, $"/catalog/item/{item.Category.Id}{{?pageSize,page}}" will result in this ->
`/catalog/item/123?12,2` which is not a valid url as `,` preferred to be url encoded.
In general I'd use something simple like query parameters ?pagesize=10&page=2
https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design

Howerer I like that `Href` uses only `path` not complete uri as per HATEOS, makes way simple to make calls as it becomes host independent.